awsim_sensor_kit_launch/launch/lidar.launch.xml

```diff
@@ -9,27 +11,49 @@
 
     <group>
       <push-ros-namespace namespace="top"/>
-      <include file="$(find-pkg-share common_awsim_sensor_launch)/launch/velodyne_lidar.launch.xml">
+      <include file="$(find-pkg-share common_awsim_sensor_launch)/launch/velodyne_VLP16.launch.xml">
+        <arg name="max_range" value="250.0"/>
+        <arg name="sensor_frame" value="velodyne_top"/>
+        <arg name="sensor_ip" value="127.0.0.1"/>
+        <arg name="host_ip" value="$(var host_ip)"/>
+        <arg name="data_port" value="2368"/>
+        <arg name="scan_phase" value="300.0"/>
+        <arg name="launch_driver" value="$(var launch_driver)"/>
```

Comment:
@SakodaShintaro 
Don't we need cloud_max_angle and cloud_min_angle parameters?

Reply:
@shmpwk @SakodaShintaro 
Nebula just underwent a major change, an Max tells me it would be better to upgrade it.
I will do it (unless someone else offers themselves) but I am behind in several projects so I can not give you a timeline.

In the new nebula, all parameters are needed, but some will probably fail since this is a simulated sensor... (needs to be checked)

Reply:
@knzo25 
Thank you for your reply.
At the moment, it is okay that only the version specified in autoware.repos:v0.0.1 is supported.
https://github.com/autowarefoundation/autoware/blob/main/autoware.repos#L100-L103

I think that the default values are used for `cloud_max_angle=360` and `cloud_min_angle=0`, but are these values appropriate?
https://github.com/RobotecAI/awsim_sensor_kit_launch/pull/19/files#diff-a39c79fc788ef13c0bcee53966e163fdcea90fe23fbd162031d11e3a11156630R354-R355

Reply:
Yes, those values should do the trick

Reply:
Got it, thank you for your check!